docs/CONFIGURATION.md

```diff
@@ -85,6 +80,16 @@ Specify the thresholds for mutation scores.
 
 Setting `break=0` (default value) ensures that the build will never fail.
 
+## Process runner config
+#### test-runner-command
+
+**Config file:** `test-runner-command: { command: "sbt", args: "test" }`    
```

Comment:
As not to have *too* big of a breaking change, and to keep in line with Stryker. I think we should call this `command-runner`, or `test-runner`